std/container/slist.d

```diff
@@ -1,5 +1,5 @@
 /**
-This module implements a singly-linked list container.
+This module implements a singly-linked list container (aka stack).
```

Comment:
A linked list isn't the same thing as a stack. It can be *used* to implement a stack; maybe that's what we should put here.